src/pages/overview/document-generation-api/templatetags.md

```diff
@@ -245,6 +245,32 @@ Default extension is vertical. If cell-extension construct is not provided, then
 
 ![Table gets horizontally extended as table columns are repeated](../images/horizontal-extension1.png)
 
+### Add index based conditions
```

Comment:
Index based conditions are only supported inside tables. Since the section _Add index-based conditions_ is already present as a subsection of Table Tags, I've not added table in the section heading.